src/main/java/com/conveyal/datatools/manager/jobs/MonitorServerStatusJob.java

```diff
@@ -87,131 +91,135 @@ public void jobLogic() {
         // Get OTP URL for instance to check for availability.
         boolean routerIsAvailable = false, graphIsAvailable = false;
         // If graph was not already built by a previous server, wait for it to build.
-        if (!graphAlreadyBuilt) {
-            boolean bundleIsDownloaded = false;
-            // Progressively check status of OTP server
-            if (deployment.buildGraphOnly) {
-                // No need to check that OTP is running. Just check to see that the graph is built.
-                bundleIsDownloaded = true;
-                routerIsAvailable = true;
+        try {
```

Comment:
Does this whole block really need to be wrapped in the try/catch? Could we localize this a little more to where the instance health check happens?